src/GraphQL.Conventions/Adapters/Engine/GraphQLEngine.cs

```diff
@@ -263,7 +263,7 @@ public GraphQLExecutor NewExecutor(IRequestDeserializer requestDeserializer = nu
 
         public string SerializeResult(ExecutionResult result)
```

Comment:
Consider to make this method async as well.

Reply:
Added a SerializeResultAsync method. Kept the non-async one for backward compatibility.

Reply:
see #195